tests/test_transform_probes.py

```diff
@@ -133,6 +133,54 @@
     }
 }
 
+OUT_PROBE_DATA_BY_CHANNEL = {
+    'release': {
+        'histogram/TEST_HISTOGRAM_1': {
+            'history': [
+                {
+                    'cpp_guard': None,
+                    'description': 'A description.',
+                    'details': {
+                        'high': 10,
+                        'keyed': False,
+                        'kind': 'exponential',
+                        'low': 1,
+                        'n_buckets': 5
+                    },
+                    'expiry_version': '53.0',
+                    'optout': True,
+                    'revisions': {
+                        'first_revision': 'node_id_2',
+                        'first_version': '51',
```

Comment:
Can we add them as `versions: { first, last }` and leave `revisions` as is?
Then we can also add these to the "all channels output" and have the generally same format (except for the history layout).
probe-dictionary can soft-migrate then, we wouldn't need any special handling.

Reply:
@Dexterp37 no, I think that's a good idea, it will make it easier to parse and use; but it does then pose the question: why have the monolithic at all, since it is just equal to the sum of the channel-breakdown partrs?

Reply:
The monolithic will only be used during the transition period, so that we're not forced to change the probe dictionary

Reply:
We can revisit this in the future, but for e.g. the probe-dictionary i usually need to pull all the channels.
Having this all in one file does:
- limit the number of http requests (perf impact)
- probably improves compression ratio (perf impact)
- generally reduces duplicate data (some probe data is shared across channels)